src/actix/api/retrieve_api.rs

```diff
@@ -52,7 +55,7 @@ pub async fn get_point(
         }
     };
 
-    let response = do_get_point(toc.get_ref(), &collection_name, point_id).await;
+    let response = do_get_point(toc.get_ref(), &collection_name, point_id, todo!()).await;
```

Comment:
Should we add an optional query parameter `read_consistency` to the `get_point` API?

Reply:
yes, why not?

Reply:
Done.